api/v1alpha1/contentsourcebinding_types.go

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2020 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ContentSourceReference contains info to locate a Kind ContentSource object
+type ContentSourceReference struct {
+	// API version of the referent.
+	APIVersion string `json:"apiVersion,omitempty"`
+	// Kind is the type of resource being referenced.
+	Kind string `json:"kind,omitempty"`
+	// Name is the name of resource being referenced.
+	Name string `json:"name"`
+}
+
+// +kubebuilder:object:root=true
+// +kubebuilder:resource:scope=Namespaced,shortName=contentsourcebinding
+// +kubebuilder:printcolumn:name="ContentSource",type="string",JSONPath=".contentSourceRef.name"
+
+// ContentSourceBinding is a binding object responsible for
+// defining a ContentSource and a Namespace associated with it
+type ContentSourceBinding struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	// ContentSourceRef is a reference to a ContentSource object
+	ContentSourceRef ContentSourceReference `json:"contentSourceRef,omitempty"`
+}
+
+// +kubebuilder:object:root=true
+
+// ContentSourceBindingList contains a list of ContentSourceBinding
```

Comment:
`ContentSourceBinding` -> `ContentSourceBindings`?

Reply:
The format of this string is how kubebuilder would have generated it, and we've kept it that way (non-plural) in the other files here.

Reply:
Resolving this conversation as per Bryan's comment.